src/cursor.ts

```diff
@@ -41,30 +45,42 @@ class Cursor extends Point {
     options: CursorOptions,
     controller: Controller
```

Comment:
Make `private` and remove the members and assignment in the constructor.

Reply:
Addressed in 5623ddee71633c0331510c41f9cced49a2dfe07f